src/include/daos_obj.h

```diff
@@ -403,6 +413,9 @@ daos_obj_punch(daos_handle_t oh, daos_handle_t th, daos_event_t *ev);
  *			-DER_UNREACH	Network is unreachable
  *			-DER_EP_RO	Permission denied
  *			-DER_NOEXIST	Nonexistent object ID
+ *			-DER_EP_OLD	Related RPC is resent too late as to
+ *					related DTX may has been aggregated.
```

Comment:
DTX is too specific for user, you might want to describe it in high level

Reply:
Original I thought we should return -DER_TIMEDOUT under such case, but during the test I just realized that the -DER_TIMEDOUT will cause the client to retry the RPC again and again. So I have to use another special error code, the DER_EP_OLD is relative suitable for that. I will update the comment and NOT mention DTX related things.